apps/graphql/src/apps/itinerary/helpers/stopoverDuration.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import differenceInMinutes from 'date-fns/differenceInMinutes';
+
+import GraphQLSector from '../../common/types/outputs/Sector';
+
+const getStopoverDuration = (
+  sector: GraphQLSector, // TODO: Check type
+  previousSector: ?GraphQLSector,
+): number | null => {
```

Comment:
Is there any difference between `number | null` and `?number`?

Reply:
Yes, number | null means it can be a number or null while ?number means it can be number, null or undefined

Reply:
Just noticed, I wanted to improve that type, so making it WIP again.

Reply:
Yes, that type was bad. Improved.